examples/README.md

```diff
@@ -155,6 +155,7 @@ Example | File | Description
 
 Example | File | Description
 --- | --- | ---
+`ambiguity_checker` | [`ecs/ambiguity_checker.rs`](./ecs/ambiguity_checker.rs) | Shows how the ambiguity checker works
 `ecs_guide` | [`ecs/ecs_guide.rs`](./ecs/ecs_guide.rs) | Full guide to Bevy's ECS
```

Comment:
In my opinion, the `ecs_guide` example should still be the first entry in that Table.
While the vast majority of the Tables are sorted alphabetically, the `ecs_guide` is significant/foundational enough to be placed first here.

Reply:
I agree with that, but I've just followed the convention. If more folks agree with that change, I'll do this quick and easy.

Reply:
I agree: I'd like to leave the `ecs_guide` at the top.

Reply:
Done in https://github.com/bevyengine/bevy/pull/2766/commits/1967b7b79c3d9f62b23afc439d3089ea8fc17f95